src/cf-typography/usage.md

```diff
@@ -566,12 +537,12 @@ It is primarily used within a max-width 599px media query (see `.jump-link` and
 
 #### Default example
 
-<a class="block-link" href="#">
+<a class="a-block-link" href="#">
     Default block link
 </a>
 
 ```
-<a class="block-link" href="#">
+<a class="a-block-link" href="#">
```

Comment:
All of the link examples in this file are not using their full updated class names.